varcode/variant_collection.py

```diff
@@ -110,49 +67,55 @@ def __str__(self):
     def __repr__(self):
         return str(self)
 
-    def clone(self, new_variants=None):
+    def clone_metadata(self, new_variants):
         """
         Create copy of VariantCollection with same metadata but possibly
-        different Variant entries. If no variants provided, then just make a
-        copy of self.variants.
+        different Variant entries.
         """
-        if new_variants is None:
-            new_variants = self.variants
+
         return VariantCollection(
-            variants=list(new_variants),
-            original_filename=self.original_filename,
-            reference_path=self.reference_path,
-            reference_name=self.reference_name,
-            ensembl_release=self.ensembl_release)
+            variants=new_variants,
+            original_filename=self.original_filename)
+
+    def clone(self):
+        return self.clone_metadata(list(self.variants))
 
     def drop_duplicates(self):
         """
         Create a new VariantCollection without any duplicate variants.
         """
-        return self.clone(set(self.variants))
+        return self.clone_metadata(set(self.variants))
 
-    def variant_effects(self, raise_on_error=True):
+    def annotations(self, raise_on_error=True):
```

Comment:
this method may be unnecessary? I'm fine to call .annotation myself on each variant in a list